src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -7,7 +7,7 @@
 
 @pytest.fixture(scope='function')
 def session():
-    with nifgen.Session('', False, 'Simulate=1, DriverSetup=Model:5433 (2CH);BoardType:PXIe') as simulated_session:
+    with nifgen.Session('', '0', False, 'Simulate=1, DriverSetup=Model:5433 (2CH);BoardType:PXIe') as simulated_session:
```

Comment:
Need to test where channel isn't passed (use default) and where None is passed.

Reply:
Done